pkg/wamr/Makefile

```diff
@@ -0,0 +1,61 @@
+PKG_NAME=wamr
+PKG_URL=https://github.com/bytecodealliance/wasm-micro-runtime.git
+PKG_VERSION=main
```

Comment:
i do not like the commit hashes at least not while developing and or the package is maintained (in debian speak) they lead to a situation like libcoap which is 8 years out of date

Reply:
But this is a moving target, one CI build might succeed, and the a PR completely unrelated might break because the latest changes in `wamr` break something in our integration.

Reply:
we allready got an example there is change in riot the needs a upstream change (or patch-file but no one really like these)

Reply:
This has to be fixed to a given commit, otherwise this will be a nightmare to maintain.

The completely outdated `libcoap` package isn't because the version was fixed to a specific commit, but because nobody cared to maintain the package. If the package would have followed `master`, it would have caused breaking CI all the time. This in turn would have pissed people of to either nail down the package version to a specific commit (same result) or just drop the package.

The only way to prevent code from bit-rotting is to get people maintaining it.

Reply:
defaulting to a specific commit now